### PR TITLE
ConnectionPool: reconnect immediately after a recv/send error

### PR DESCRIPTION
**Pull request by mckelvin**
An addition to #11 

cc @zzl0 @youngsofun

### CODE REVIEW

File: libmc/__init__.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 __version__ = "v0.5.2"
```
Comment (zzl0) [inline libmc/__init__.py]: need to change version ?

Comment (mckelvin) [inline libmc/__init__.py]: @zzl0 Not now. I will bump version after it is stable.

Comment (zzl0) [inline libmc/__init__.py]: ok

**Comment by zzl0** [comment]
lgtm

**Pull request merged**